packages/neuron-wallet/src/database/chain/entities/sync-progress.ts

```diff
@@ -0,0 +1,48 @@
+import { HexString } from '@ckb-lumos/base'
+import { scriptToHash } from '@nervosnetwork/ckb-sdk-utils'
+import { Entity, PrimaryColumn, Column } from 'typeorm'
+
+@Entity({ name: 'sync_progress' })
+export default class SyncProgress {
+  @PrimaryColumn({ type: 'varchar' })
+  hash!: string
+
+  @Column({ type: 'varchar' })
+  args!: string
+
+  @Column({ type: 'varchar' })
+  codeHash!: string
+
+  @Column({ type: 'varchar' })
+  hashType!: CKBComponents.ScriptHashType
+
+  @Column()
+  scriptType!: CKBRPC.ScriptType
+
+  @Column({ type: 'varchar' })
+  walletId!: string
+
+  @Column()
+  blockStartNumber: number = 0
+
+  @Column()
+  blockEndNumber: number = 0
+
+  @Column({ type: 'varchar' })
+  cursor?: HexString
+
+  @Column({ type: 'boolean' })
+  softDelete: boolean = false
```

Comment:
Not sure if `deleted` is better, I think any `delete` field in the table is `soft`